easy_toolbox.py

```diff
@@ -110,6 +109,7 @@ def get_action_from_args() -> Option:
 
 
 def get_action_from_cli() -> Option:
```

Comment:
actually this is a good moment to fix the function name: `get_action_from_cli` -> `get_action_from_gui`

Reply:
done